cmake/configs/nuttx_px4fmu-v3_rtps.cmake

```diff
@@ -0,0 +1,16 @@
+include(configs/nuttx_px4fmu-v3_default)
+include(common/rtps)
+
+list(APPEND config_module_list
+	# micro RTPS
+	modules/micrortps_bridge/micrortps_client
+	lib/micro-CDR
+)
+
+set(config_rtps_send_topics
```

Comment:
Can we move the set of topics to `common/rtps`? Will be easier to manage.

Reply:
That sounds fine to me, but I don't think we know how this is actually intended to be used.